NSTTutorials/src/test/java/com/nst/tutorials/rest/runtimearguments/README.md

```diff
@@ -0,0 +1,73 @@
+[_Return to Tutorials Overview_](https://github.com/eBay/NSTSuite/tree/main/NSTTutorials)
+
+# Runtime Arguments
+
+### Overview
+
+NST has several runtime arguments that are supported by default. In this section we will review these runtime arguments, how to add your own custom runtime arguments, and how to add your own custom logger / format writer files.
+
+To see a list of the NST runtime arguments supported out of the box, please see [here](../../../../../../../../../NST/README.md#Runtime Arguments).
+
+### Topics Covered
+
+1. [Adding custom runtime arguments](#Adding custom runtime arguments)
+2. [Adding a custom logger](#Adding a custom logger)
+3. [Overriding an existing runtime argument](#Overriding an existing runtime argument)
+
+### References
+- [RuntimeConfigManager](../../../../../../../../../NST/src/main/java/com/ebay/runtime/RuntimeConfigManager.java)
+
+## Adding custom runtime arguments
```

Comment:
Before we launch into custom runtime arguments we want to talk about how we specify them. Both as runtime arguments in the command line, but also as arguments to the surefire plugin in the pom. Discuss the pros/cons.